src/vsr/replica.zig

```diff
@@ -3477,9 +3622,25 @@ pub fn ReplicaType(
             }
         }
 
-        fn on_request_read_reply_callback(client_replies: *ClientReplies, reply_: ?*Message) void {
+        fn on_request_read_reply_callback(
```

Comment:
NB: we have 

```
fn on_request_reply_read_callback
fn on_request_read_reply_callback
```

Reply:
Maybe factor out the whole on_request_ignore_resend logic into a separate function, so that we:

* don't do send directly from ignore, which feels a bit weird
* have a name for this flow?